REVIEW_GUIDELINES.md

```diff
@@ -279,7 +279,7 @@ ingress:
 
 ```yaml
 {{- if .Values.ingress.enabled -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
How about this to make it compatible with versions <= k8s 1.13

```suggestion
{{ if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
apiVersion: networking.k8s.io/v1
{{ else }}
apiVersion: extensions/v1beta1
{{ end -}}
```

Reply:
I wonder if it would be fine to recommend `networking.k8s.io/v1beta1` in the guidelines to avoid the added complexity of the conditional? Chart owners can decide whether or not to support older versions of k8s, but maybe we shouldn't mention deprecated API versions in the guidelines? (Just a philosophical question, no strong opinion here.)

Reply:
Given that a bunch of folks are still on older versions of k8s, I take back my suggestion. Once old versions become _really_ old, then I'm sure documentation will be updated. (I'll let you resolve the conversation when you are OK with it, @lachie83.)